PIPs/pip-31.md

```diff
@@ -12,67 +12,84 @@ created: 08-08-2024
-The default value for `fixed_fee` is proposed to be set to zero.
-
 #### Consumption
 
 To understand the Consumptional Fee Model, we first need to define consumption.
 
-Transactions are sequences of bytes that are decoded, processed, and stored by validators.
-Consumption is defined as the number of bytes stored over the last 8640 blocks [^1] for a specific account.
+Transactions are sequences of bytes that validators decode, process, and store in their local database.
+Consumption is defined as the number of bytes stored over the last 8640 blocks [^1]
+for a specific account that signed a transactions.
 
 #### Coefficient
 
-The coefficient is a unitless number that starts at zero and grows exponentially: $0, 1, 2, 4, 8, \dots $
+The coefficient is a unitless number that starts at zero and grows naturally: $0, 1, 2, 3, \dots $
 
 The coefficient would be calculated as follows:
 
 $$
-n = \frac{\text{consumption}}{\text{daily_limit}}
+\text{coefficient} = \lfloor \frac{\text{consumption}}{\text{daily_limit}} \rfloor
 $$
 
-$$
-\displaylines{\text{coefficient} = \begin{cases}  0 & \text{if } n = 0 \\ 2^{n-1} & \text{if } n \geq 1 \end{cases}
-}
-$$
+In this formula [^2], the daily limit is the number of bytes an account can send each day without paying a fee.
+This parameter is part of the node configuration, allowing each validator to set their preferred value.
 
-In this model, the daily_limit is the number of bytes an account can send each day without paying a fee.
-This parameter would be part of the node configuration, allowing each validator to set their preferred value.
-
-The default value for `daily_limit` is proposed to be **300 bytes**
+The only exception here is the account that broadcasts its first transaction.
+The coefficient for an account that sends its first transaction is always set to `1`.
 
 #### Unit Price
 
-The unit_price would define the fee per byte in PAC. This parameter would be part of the node configuration,
-allowing each validator to set their preferred value.
+The unit price would define the fee per byte in PAC.
+This parameter is part of the node configuration, allowing each validator to set their preferred value.
+
+### Configuration
 
-The default value for `unit_price` is proposed to be **0.0001 PAC**.
+All parameters for calculating the fee are configurable.
+However, we recommend the following configurations:
 
-### Account Creation Fee
+For most validators, we suggest setting the daily limit to `0 bytes` and the fixed fee to `0.01 PAC`.
+This configuration eliminates the consumption fee, meaning these validators will only apply the fixed fee.
 
-Each account needs to pay a `fixed_fee` the first time, after which a consumptional fee is applied.
+For validators with good resources, we recommend setting the daily limit to `280 bytes`,
+the unit price to `0.000005 PAC`, and the fixed fee to `0 PAC`.
+This allows users to send almost up to 5 free or low-fee transactions,
```

Comment:
I think 5 transactions free is high, better between 1~2 txs per day for free.

Reply:
Based on these parameters almost 2 is free and 3 has fee less than `0.01 PAC`.